packages/client/src/modules/play/context/playContext.tsx

```diff
@@ -327,6 +328,9 @@ function useGameSocket({
     newSocket.on("gameUpdated", ({ update }: { update: Partial<IGame> }) => {
       setGameWithTeams((previous) => {
         if (previous === null) return null;
+        if (previous.status !== "finished" && update.status === "finished") {
```

Comment:
Histoire de bien comprende, `STEPS.length - 2` est l'étape de synthèse et `STEPS.length - 1` de `finish` ? Comme à l'étape `STEPS.length - 1` on indique que le jeu est `finish,` on risque de skipper l'étape de synthèse avec la redirection de la ligne du dessous

Reply:
Non c'est STEPS.length - 1 qui est l'étape de synthèse on n'a pas d'étape de finish, l'étape de synthèse se passe et quand on termine l'étape on passe `isStepActive` à false et on dit que le jeu est `finish` et ça redirige les joueurs

Reply:
@Baboo7 je merge mais si tu vois un truc à changer je peux le changer dans une prochaine PR

Reply:
Got it!